pkg/api/annotations.go

```diff
@@ -600,22 +600,11 @@ func AnnotationTypeScopeResolver(annotationsRepo annotations.Repository, feature
 			Permissions: map[int64]map[string][]string{
 				orgID: {
 					dashboards.ActionDashboardsRead:     {dashboards.ScopeDashboardsAll},
-					accesscontrol.ActionAnnotationsRead: {accesscontrol.ScopeAnnotationsAll},
+					accesscontrol.ActionAnnotationsRead: {accesscontrol.ScopeAnnotationsTypeOrganization, accesscontrol.ScopeAnnotationsAll},
```

Comment:
@IevaVasiljeva is this change ok? I don't really know if we have to care about the feature toggle

Reply:
See https://github.com/grafana/grafana/pull/102612/commits/cb761b0acb16033fc6571a5f1b054f93f438c7f2